File: twoCube/Entities/Survey.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using NHibernate;
using NHibernate.Criterion;

namespace twoCube.Entities
{
    public class Survey
    {
        public virtual int Id { get; set; }
        public virtual string surveyTitle { get; set; }
        public virtual string surveyDescription { get; set; }
        public virtual bool surveyStatus { get; set; }
        public virtual DateTime surveyCreated { get; set; }
        public virtual DateTime surveyStartDate { get; set; }
        public virtual DateTime surveyEndDate { get; set; }
        public virtual IList<SurveyQuestion> surveyQuestionList { get; set; }
        public virtual IList<Respondent> respondentList { get; set; }
        public Survey()
        {
            surveyQuestionList = new List<SurveyQuestion>();
            respondentList = new List<Respondent>();
            surveyCreated = DateTime.Now;
        }

        public virtual void AddSurveyQuestion(SurveyQuestion surveyQuestion)
        {
            surveyQuestionList.Add(surveyQuestion);
        }

        public virtual void AddRespondent(Respondent respondent)
        {
            respondentList.Add(respondent);
        }
        public static Survey GetById(ISession session, int id)
	        {
                return session.CreateCriteria(typeof(Survey))
	                .Add(Expression.Eq("Id", id))
                    .UniqueResult<Survey>();
	        }

    }
}

File: twoCube/src/com/twocube/entities/Survey.java
package com.twocube.entities;

import java.util.ArrayList;
import java.util.List;

public class Survey {
	private List<SurveyQuestion> questionList;

	public Survey(){
		this.questionList = new ArrayList<SurveyQuestion>();
	}
	public Survey(List<SurveyQuestion> questionList) {
		this.questionList = questionList;
	}

	public List<SurveyQuestion> getQuestionList() {
		return questionList;
	}

	public void setQuestionList(List<SurveyQuestion> questionList) {
		this.questionList = questionList;
	}
	
}
